conf/modules/rank_variants.config

```diff
@@ -37,14 +37,20 @@ process {
         ext.prefix = { "${meta.id}_ranked_sv" }
     }
 
-    withName: '.*RANK_VARIANTS_SV:TABIX_BGZIPTABIX' {
+    withName: '.*RANK_VARIANTS_SV:BCFTOOLS_SORT' {
+        ext.args = "--output-type z"
         ext.prefix = { "${meta.id}_ranked_sv" }
+        ext.when = true
```

Comment:
As far as I can tell, it should work as intended with explicitly setting `ext.when` to `true`.

Reply:
Yupe! You are right, I will remove it in the next commit. 👍🏻

Reply:
It seems you haven't removed the `ext.when = true`  ;-)

Reply:
Done! 👍🏻 :)